src/dir.targets

```diff
@@ -172,4 +172,27 @@
 
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
+
+  <!-- Generate coverage for individual projects. -->
+  <PropertyGroup>
+    <ProjectInput>$(MSBuildProjectName).dll.coverage.xml</ProjectInput>
+  </PropertyGroup>
+  
+  <Target Name="GenerateIndividualCoverageReport"
+    AfterTargets="RunTestsForProject"
+    Inputs="$(CoverageReportDir)\$(ProjectInput)"
+    Outputs="$(CoverageReportDir)$(MSBuildProjectName)_*.*"
```

Comment:
I would need to verify this but if I recall wildcards aren't supported as Outputs so the incremental build for this will be broken.